index.html

```diff
@@ -2,261 +2,341 @@
-                </li>
-
-                <li class="icon peace">
-                    <a href="https://www.jw.org/en"> <img src="./Graphics/Vector.svg" alt="peace"></a>
-                </li>
-            </ul>
-        </section>
-
-        <section class="work">
-            <ul>
-                <li>
-                    <img class="portfolio one" src="./Graphics/Snapshoot Portfolio.jpg" alt="">
-                    <h1 class="projectTitle tonic">Tonic</h1>
-
-                    <ul class="subtitle tonic">
-                        <li class="sub canopy">CANOPY </li>
-                        <li><img src="./Graphics/Counter.svg" alt="counter"></li>
-                        <li class="sub backend">Back End Dev</li>
-                        <li><img src="./Graphics/Counter.svg" alt="counter"></li>
-                        <li class="sub year">2015</li>
-                    </ul>
-
-                    <p class="work description">A daily selection of privately personalized
-                        reads; no accounts or sign-ups required.
-                    </p>
-
-                    <ul class="work madewith">
-                        <li class="html">html</li>
-                        <li class="css">css</li>
-                        <li class="js">javascript</li>
-                    </ul>
-
-                    <div class="work seeproject">See Project</div>
-                </li>
-            </ul>
-            <div class="space"></div>
-
-            <ul>
-                <li>
-                    <img class="portfolio two" src="./Graphics/Snapshoot Portfolio 2.jpg" alt="">
-                    <h1 class="projectTitle multi">Multi-Post Stories</h1>
-
-                    <ul class="subtitle tonic">
-                        <li class="sub canopy">CANOPY </li>
-                        <li><img src="./Graphics/Counter.svg" alt="counter"></li>
-                        <li class="sub backend">Back End Dev</li>
-                        <li><img src="./Graphics/Counter.svg" alt="counter"></li>
-                        <li class="sub year">2015</li>
-                    </ul>
-
-                    <p class="work description">A daily selection of privately personalized
-                        reads; no accounts or sign-ups required.
-                    </p>
-
-                    <ul class="work madewith">
-                        <li class="html">html</li>
-                        <li class="css">css</li>
-                        <li class="js">javascript</li>
-                    </ul>
-
-                    <div class="work seeproject">See Project</div>
-                </li>
+    <header class="elements">
+      <p><a href="https://github.com/calebchris000" class="logo">CALEB</a></p>
+      <nav class="hamburger"><img src="./Graphics/Normal Button/Tertiary/Icons/Union.svg" alt=""></nav>
+      <ul>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
```

Comment:
- [ ] Kindly make these `a` tags where in each tag we have `href` with the id of the relevant section, that way your links will be functional and they'll take us to the right sections 👍 

```
<li><a href="#portfolio">Portfolio</a></li>
```